Cargo.toml

```diff
@@ -71,6 +71,8 @@ url = ">= 2.1, < 3.0"
 uuid = { version = ">= 0.8, < 2.0", features = ["v4"] }
 walkdir = "^2.3"
 xz2 = "^0.1"
+#nmstate = "2.1.0"
+nmstate = { path = "../nmstate/rust/src/lib" }
```

Comment:
This looks like leftover from testing.

Reply:
The nmstate rust lib is not published yet at cargo.io they are going to do so during this week I think. Meanwhile I point it there to my local clone, this is draft PR.